tests/TestCase/Event/EventManagerTest.php

```diff
@@ -811,13 +811,30 @@ public function testDebugInfo()
             [
                 '_listeners' => [],
                 '_isGlobal' => false,
-                '_eventList' => null,
                 '_trackEvents' => false,
                 '_generalManager' => '(object) EventManager',
+                '_dispatchedEvents' => null,
```

Comment:
Why the key rename?

Reply:
This is the key `__debugInfo()` meant to set when event list is not empty but never got set due to buggy code. I can change it to `_eventList` if you like.

Reply:
No this is fine, I was just confused.